lib/ft817/ft817.cpp

```diff
@@ -399,9 +400,10 @@ void FT817::getBytes(byte count)
 	while (rigCat.available() < 1 && millis() < startTime + 2000) { ; }
 
 	flushBuffer();
-	for (byte i=count; i>0; i--)
+	for (byte i=0; i<count; i++)

```

Comment:
This is where the received bytes from the radio were being loaded into the buffer array the wrong away around